src/display/annotation_layer.js

```diff
@@ -297,6 +297,110 @@ class AnnotationElement {
     return container;
   }
 
+  get _commonActions() {
```

Comment:
All of the code in this method is just copy-pasted from its previous position, so I don't really have to "review it" here, right?

Reply:
Yes no need to review it.